versions/3.0.3.md

```diff
@@ -71,7 +71,9 @@ An OpenAPI definition can then be used by documentation generation tools to disp
 A document (or set of documents) that defines or describes an API. An OpenAPI definition uses and conforms to the OpenAPI Specification.
 
 ##### <a name="pathTemplating"></a>Path Templating
-Path templating refers to the usage of curly braces ({}) to mark a section of a URL path as replaceable using path parameters.
+Path templating refers to the usage of variable expressions, delimited by curly braces ({}), to mark a section of a URL path as replaceable using path parameters.
```

Comment:
It would be good if we could align with the RFC6570 terminology and call these things template expressions.
```suggestion
Path templating refers to the usage of template expressions, delimited by curly braces ({}), to mark a section of a URL path as replaceable using path parameters.
```